lib/debuglet.js

```diff
@@ -215,7 +217,18 @@ Debuglet.prototype.scheduleBreakpointFetch_ = function(seconds) {
 
         default:
           that.logger_.info('\t' + response.statusCode + ' completed.');
-          that.updateActiveBreakpoints_(body.breakpoints || []);
+          var bps = (body.breakpoints || []).filter(function(bp) {
+            var action = bp.action || 'CAPTURE';
+            if (action !== 'CAPTURE') {
+              that.logger_.warn('Found breakpoint with invalid action:', action);
```

Comment:
I think `rejectBreakpoint` should be fully general. Currently `completeBreakpoint` also relies on the status being set outside the function since the status requires information about the call site.